IpmiFeaturePkg/Test/Mock/Include/GoogleTest/Library/MockIpmiCommandLib.h

```diff
@@ -0,0 +1,345 @@
+/** @file MockIpmiCommandLib.h
+  Google Test mocks for IpmiCommandLib
+
+  Copyright (c) Microsoft Corporation.
+  Your use of this software is governed by the terms of the Microsoft agreement under which you obtained the software.

```

Comment:
This is an open source repo, this needs to be 

  Copyright (c) Microsoft Corporation
  SPDX-License-Identifier: BSD-2-Clause-Patent